Makefile

```diff
@@ -7,11 +7,19 @@ build:
 	cargo build --release
 
 .PHONY: test
-test:
-	RUST_LOG=$(LOG_LEVEL) cargo test --all --no-fail-fast -- --nocapture --include-ignored
+test: test-unit test-integration
```

Comment:
Let's also break out the clippy and fmt checks into their own target, maybe `lint`, then `test` can run these first (eg `test: lint test-unit test-integration`)

Reply:
Updated it. Thank you.